upgrades/upgrade_testing_framework/cluster_management/node_configuration.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict, List
+
+import upgrade_testing_framework.core.versions_engine as ev
+
+def _get_engine_data_dir_for_version(version: ev.EngineVersion) -> str:
+    if ev.ENGINE_ELASTICSEARCH == version.engine:
+        return "/usr/share/elasticsearch/data"
+    return "/usr/share/opensearch/data"
+
+def _get_engine_user_for_version(version: ev.EngineVersion) -> str:
+    if ev.ENGINE_ELASTICSEARCH == version.engine:
+        return "elasticsearch"
+    return "elasticsearch"
```

Comment:
this should be `return "opensearch"`, yeah?

Reply:
Surprisingly not; for Elastic 1.x my testing indicated the run-as user in the container was "elasticsearch".  Snapshot/restore certainly seems to work with the OpenSearch container's snapshots volume having its linux ownership perms set to the "elasticsearch" user.

Reply:
well that's weird (but as a side note, this explains why the original snapshot walkthrough only needed the owner of the repo to be set once).

why have the `if` at all then?

Reply:
Because I'm betting OpenSearch 2.x+ has a different user, and this is looking ahead to that eventuality.